include/nbl/asset/interchange/IAssetLoader.h

```diff
@@ -301,6 +337,50 @@ class IAssetLoader : public virtual core::IReferenceCounted
 	{
 		bundle.setAsset(offset,std::move(_asset));
 	}
+
+private:
+	static IAssetLoader::SAssetLoadParams getFindParams(IAssetLoader::SAssetLoadParams _params)
+	{
+		_params.cacheFlags = static_cast<E_CACHING_FLAGS>(_params.cacheFlags & ECF_DONT_CACHE_REFERENCES);
+		return _params;
+	}
+	SAssetBundle interm_getAssetInHierarchy_find(IAssetManager* _mgr, io::IReadFile* _file, const std::string& _supposedFilename, const IAssetLoader::SAssetLoadParams& _params, uint32_t _hierarchyLevel, IAssetLoader::IAssetLoaderOverride* _override)
+	{
+		return interm_getAssetInHierarchy(_mgr, _file, _supposedFilename, getFindParams(_params), _hierarchyLevel);
+	}
+	SAssetBundle interm_getAssetInHierarchy_find(IAssetManager* _mgr, const std::string& _filename, const IAssetLoader::SAssetLoadParams& _params, uint32_t _hierarchyLevel, IAssetLoader::IAssetLoaderOverride* _override)
+	{
+		return interm_getAssetInHierarchy(_mgr, _filename, getFindParams(_params), _hierarchyLevel, _override);
+	}
+	SAssetBundle interm_getAssetInHierarchy_find(IAssetManager* _mgr, io::IReadFile* _file, const std::string& _supposedFilename, const IAssetLoader::SAssetLoadParams& _params, uint32_t _hierarchyLevel)
+	{
+		return interm_getAssetInHierarchy(_mgr, _file, _supposedFilename, getFindParams(_params), _hierarchyLevel);
+	}
+	SAssetBundle interm_getAssetInHierarchy_find(IAssetManager* _mgr, const std::string& _filename, const IAssetLoader::SAssetLoadParams& _params, uint32_t _hierarchyLevel)
+	{
+		return interm_getAssetInHierarchy(_mgr, _filename, getFindParams(_params), _hierarchyLevel);
+	}
```

Comment:
could we do this via "perfect forwarding" or a similar template so I'm not seeing all this duplicate code